docs/getting-started/installing-with-docker.md

```diff
@@ -208,7 +207,7 @@ Docker Compose will start the container, and a collect run will begin automatica
 
 :::note
 
-By default, Resoto collects [anonymous statistics](../reference/telemetry.md) about how the product is used. However, this telemetry can be [disabled](../reference/telemetry.md#disabling) by setting the `RESOTOCORE_ANALYTICS_OPT_OUT` environment variable.
+By default, Resoto collects [anonymous statistics](../reference/telemetry.md) about how the product is used. However, this telemetry can be [disabled](../reference/telemetry.md#disabling) by setting the `resotocore.runtime.analytics_opt_out=true` config variable.
```

Comment:
Should provide a sample YAML snippet here for users editing using `configs edit`, and/or a command using `configs set` that users can simply copy/paste/execute.

Reply:
Done